src/components/Select/MultiSelect.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import ChevronDownIcon from '@moda/icons/chevron-down-12';
+import ChevronUpIcon from '@moda/icons/chevron-up-12';
+import classNames from 'classnames';
+import { SelectableOption } from '.';
+import { SelectOptions } from './SelectOptions';
+import { useSelect } from './useSelect';
+import ExitIcon from '@moda/icons/exit-16';
+import './MultiSelect.scss';
+
+type Props = Omit<React.HTMLAttributes<HTMLDivElement>, 'defaultValue' | 'value' | 'onChange'> & {
+  defaultValue?: string | undefined;
+  disabled?: boolean;
+  idRef?: string;
+  name?: string;
+  onChange?: (value: string) => void;
+  onRemoveSelectedItem: (value: string) => void;
+  options: SelectableOption[];
+  value?: string | undefined;
```

Comment:
I'd make this have `value: string[]`, `defaultValue: string[]` and `onChange: (value: string[]) => void` and then call `onChange` when you add/remove an item.